java/opendcs/src/main/java/opendcs/dai/DaiBase.java

```diff
@@ -11,8 +11,9 @@
 import opendcs.dao.DaoBase;
 import opendcs.dao.DaoHelper;
 import opendcs.util.functional.DaoConsumer;
+import org.opendcs.database.api.OpenDcsDao;
 
-public interface DaiBase extends AutoCloseable
+public interface DaiBase extends AutoCloseable, OpenDcsDao
```

Comment:
FYI, I've been intentionally not doing this; however, I see no need to change here for you're current testing.

I think the main goal should be to not have the DAI/DAO chain of inheritance but it's going to take time to break.

Good news is, on the rest_api end you shouldn't need to do anything. Just not going to merge in in to the target so it's clear which DAOs have actually been migrated.